theano/gof/tests/test_opt.py

```diff
@@ -572,6 +572,8 @@ def test_2(self):
         assert str(g) == '[Op2(x, y)]'
 
     def test_low_use_ratio(self):
+        backup = theano.config.on_opt_error
+        theano.config.on_opt_error = 'warn'
```

Comment:
I mean this test. Which value of max_use_ratio is needed to don't have the error/warning?